docs/troubleshooting-oauth2.md

```diff
@@ -41,25 +41,32 @@ OAuth resource [rabbitmq] not available. OpenId Discovery endpoint https://<the_
 #### Troubleshoot the issue
 
 These are the most common reasons for this issue:
-- The endpoint is either down or is not reachable (e.g. there is a firewall which blocks access).
-- The endpoint has a SSL certificate not trusted by the browser. 
-- The browser is blocking access due to [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) policy.
 
-The quickest way to identity the root cause is by opening the browser's javascript console and searching for `net::ERR_`. These are the two possible errors you may find:
-- `net::ERR_CONNECTION_REFUSED` The endpoint is down or is not reachable. 
-- `net::ERR_CERT_AUTHORITY_INVALID` The endpoint's SSL Certificate is not trusted by your browser. If you want to trust this certificate, click on the URL in the error message. The browser will prompt you to trust it. 
+ * The endpoint is either down or is not reachable (e.g. there is a firewall which blocks access)
+ * The endpoint has a TLS certificate not [trusted](./ssl#peer-verification) by the browser
+ * The browser is blocking access due to [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) policy
 
-If you did not find any errors searching for `net::ERR` search instead for `CORS`. If you find an error similar to the one shown below, the browser is blocking the response returned by the endpoint and therefore it is not being delivered it to the management UI. This is due to the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) policy. You should ask the administrator of your Identity Provider to add the management UI's URL to the list of allowed **origins**.
+The quickest way to identity the root cause is by opening the browser's javascript console and searching for `net::ERR_`.
+These are the two most likely errors:
 
-  `Access to fetch at 'https://<the_issuer_url>>/.well-known/openid-configuration' from origin 
-  '<rabbitmq_url_to_management_ui>' has been blocked by CORS policy`. 
+ * `net::ERR_CONNECTION_REFUSED`: the endpoint is down or is not reachable
+ * `net::ERR_CERT_AUTHORITY_INVALID`: the endpoint's TLS certificate is not [trusted](./ssl#peer-verification) by the browser. To trust the certificate,
+   click on the URL in the error message and follow the prompt
 
+If there are no errors found for `net::ERR` search instead for `CORS`. An error similar to the one shown below means that the browser is blocking the response returned by
```

Comment:
rephrase to " If there are no errors found for `net::ERR`, next search for `CORS`."

don't use "shown below" for accessibility reasons, change to " An error similar to the followinng one means that the browser is blocking the response returned by ..."